### DISCUSSION TITLE
All properties as labels

### DISCUSSION THREAD

**Discussion by hermitty** [serilog-contrib/serilog-sinks-grafana-loki#140] [Q&A] [answered]
Is there any way of configuration which allows setting all properties as labels without indicating every single one using propertiesAsLabels?

**Comment by mishamyte** [comment, marked as answer]
Nope, this was a breaking change of v8, reasons are described [here](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/discussions/99).

Also it is not a good practice from Loki's perspective, because with a high probability you will face `high cardinality` issue.
You could use Grafana's `json` parser, as it is described in [Using LokiJsonTextFormatter and Grafana Loki json parser
](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Using-LokiJsonTextFormatter-and-Grafana-Loki-json-parser)

**Reply by hermitty** [reply]
Exactly what i was looking for, thanks :D
